service/docs/source/install.rst

```diff
@@ -5,14 +5,18 @@ Guide to Installation
 Continuous integration creates packages for a variety of Linux
```

Comment:
Should this page be retitled to _Guide to Service Installation_? If not, should we add information about installing the runtime and geopmpy?